flask_login_openerp/__init__.py

```diff
@@ -70,15 +73,22 @@ def logout(self):
             flash("You have been logout", "info")
         if 'openerp_password' in session:
             del session['openerp_password']
+        if 'openerp_user' in session:
+            del session['openerp_user']
         if self.logout_redirect_view:
-            return redirect(url_for(self.logout_redirect_view))
+            response = redirect(url_for(self.logout_redirect_view))
+            response.headers['Cache-Control'] = 'no-cache, no-store, ' \
```

Comment:
to avoid the `\` you can do the following format:
```python
response.headers['Cache-Control'] = ', '.join([
    'no-cache', 'no-store', 'must-revalidate'
])